src/types.ts

```diff
@@ -175,7 +170,7 @@ export interface RowRendererProps<TRow, TSummaryRow = unknown>
 }
 
 export interface RowsChangeData<R, SR = unknown> {
-  indexes: number[];
+  indexes: readonly number[];
```

Comment:
We generate it then pass it on to the users, it's fine to leave it mutable, users can manipulate if they want to.
```suggestion
  indexes: number[];
```